assets/css/style.css

```diff
@@ -212,3 +213,210 @@ a {
   height: 20px;
   width: 20px;
 }
+
+/* work-section */
+.works {
+  display: grid;
+  gap: 88px;
+  align-items: center;
+  margin-top: 114px;
+}
+
+.project1 {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.project1 img {
+  display: block;
+  align-self: center;
+  justify-content: center;
+  margin: auto;
+}
+
+.img {
+  /* width: 100%; */
+}
+
+.pri_text {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  margin: 0;
+}
+
+.pri_text h1 {
+  margin: 12px 0 0 0;
+}
+
+.det_text {
+  display: flex;
+  margin: 20px 0;
+}
+
+.det_text img {
+  align-self: center;
+  margin-left: 12px;
+  margin-right: 12px;
+}
+
+.canopy {
+  color: #344563;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.detail {
+  color: #7a869a;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.des_text {
+  color: #344563;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.tag_text ul {
+  list-style-type: none;
+  display: flex;
+  gap: 8px;
+  padding-left: 0;
+}
+
+.tags {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+}
+
+.button {
```

Comment:
- [ ] Please make sure to add the required button interactions (Active, Hover and Pressed) following the template guidelines.